installing/installing_gcp/installing-gcp-multiarch-support.adoc

```diff
@@ -0,0 +1,69 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-gcp-multiarch-support"]
+= Installing a GCP cluster with the support for configuring multi-architecture compute machines
+include::_attributes/common-attributes.adoc[]
+:context: installing-gcp-multiarch-support
+
+toc::[]
+
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures.
+
+[NOTE]
+====
+When there are nodes with multiple architectures in your cluster, the architecture of your image must be consistent with the architecture of the node. You need to ensure that the pod is assigned to the node with the appropriate architecture and that it matches the image architecture. For more information on assigning pods to nodes, see link:https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/[Assigning pods to nodes].
+====
+
+You can install a GCP cluster with the support for configuring multi-architecture compute machines.
+
+.Prerequisites
+
+* You installed the {oc-first}.
+* You have the {product-title} installation program and the pull secret for your cluster.
+
+.Procedure
+
+. Check that the `openshift-install` binary is using the `multi` payload by running the following command:
++
+[source,terminal]
+----
+$ ./openshift-install version
+----
++
+.Example output
+[source,terminal]
+----
+./openshift-install 4.17.0
+built from commit abc123def456
+release image quay.io/openshift-release-dev/ocp-release@sha256:123abc456def789ghi012jkl345mno678pqr901stu234vwx567yz0
+release architecture amd64
+----
```

Comment:
This example is the output of an openshift-install that uses a single payload, but users must use an openshift-install that has  `release architecture multi`.  

If openshfit-install has the multi-payload it will look like this:

```

./openshift-install 4.17.0-rc.3
built from commit dfd4c085a7210e49111fa8d6747016d78f98ecf2
release image quay.io/openshift-release-dev/ocp-release@sha256:18fed9398469b38d4f443361a3dca6db46279a7807cef95c2f5170d1a10edcef
release architecture multi
default architecture amd64
```

Any architecture can be installed, by default the installer will use the arch defined in  `default architecture`

Reply:
Updated as per the suggestion.